pkg/ui/src/views/statements/planView.tsx

```diff
@@ -16,115 +16,274 @@ import _ from "lodash";
 import React from "react";
 
 import { cockroach } from "src/js/protos";
-import { intersperse } from "src/util/intersperse";
 import IExplainTreePlanNode = cockroach.sql.IExplainTreePlanNode;
-import ExplainTreePlanNode_IAttr = cockroach.sql.ExplainTreePlanNode.IAttr;
+import IAttr = cockroach.sql.ExplainTreePlanNode.IAttr;
+import {ToolTipWrapper} from "oss/src/views/shared/components/toolTip";
 
-export function PlanView(props: {
-  title: string,
-  plan: IExplainTreePlanNode,
-}) {
-  if (!props.plan) {
-    return <div className="plan-view">
-      <h3>{props.title}</h3>
-      <p>No plan captured yet.</p>
-    </div>;
-  }
-  return <div className="plan-view">
-    <h3>{props.title}</h3>
-    <div className="plan-node">
-      <ul>
-        <li>
-          <PlanNode node={props.plan} />
-        </li>
-      </ul>
-    </div>
-  </div>;
+interface IPlanNode {
```

Comment:
Just call this `PlanNode` (and take off the `I`s from whichever other interfaces you introduce in this PR). Even though the interfaces defined in the protos all start with an `I`, it's not really conventional typescript to do that. In that case, it's just to distinguish them from the generated proto classes, (which I wish didn't exist… just the protos).

Reply:
gotcha - will remove the `I`'s for interfaces